app/Models/Configs.php

```diff
@@ -64,55 +64,49 @@ class Configs extends Model
 	/**
 	 * Sanity check.
 	 *
-	 * @param $value
+	 * @param ?string $value
```

Comment:
```suggestion
	 * @param string|null $value
```

For some reasons, my IDE only accepts `...|null` as valid PHPdoc syntax. (But id must be `?...` inside function declarations.) I don't know why but unless nobody else minds, I would stay with that.